integ-tests/src/cucumber/resources/features/dataservice/create_environment.feature

```diff
@@ -4,7 +4,7 @@
 Feature: Create environment
 
   Scenario: Create an environment
-    When I create an environment named "test"
+    When I create an environment
```

Comment:
Should we also change the describe_environment.feature to not include the environment name "test"?

Reply:
I tried only keeping a specific name in the definitions if it makes the test easier to understand, so, for example, we create an env named test and describe the env named test. One thing that makes it confusing is we need to create and env prefixed with test and not just named test so the tests can be run in parallel and not fail because of artifacts other tests have created. I'm wondering if we should say 'prefixed with test' instead of 'named test' in the definitions but then it defeats the purpose of trying to signify it's the same environment.

Reply:
Can we set up cucumber to generate us a unique prefix per test run? That way, each individual scenario execution can use the same random identifier as prefix to avoid collisions, but the test itself can just say `named: "blah"`.

Reply:
I see. I think the prefix is already different each run, since we use UUID for the prefix in the step definition? So I'll take the same pattern for describe environment as well.

Reply:
Makes sense.

Reply:
yes, @xjhe, correct. My concern is that future tests being implemented will skip the prefix since it's not specified in the definition.